ibm/service/database/resource_ibm_database.go

```diff
@@ -2837,6 +2870,39 @@ func validateGroupHostFlavor(groupId string, resourceName string, group *Group)
 	return nil
 }
 
+func validateMultitenantMemoryCpu(groupId string, resourceName string, resourceDefaults *Group, group *Group) error {
+	// TODO: Replace this with resourceDefaults.Memory.CPUEnforcementRatioCeiling when it is fixed
+	cpuEnforcementRatioCeiling := 16384
```

Comment:
When will this fix go in? Will that be before merging this PR?

Reply:
@obai-1 This fix won't go in until the next release cut and deployment, so we're looking at around May.

This PR will be merged sometime thing week. So that's why we gotta hard code the cpuEnforcementRatioCeiling for now.